src/main.ts

```diff
@@ -213,7 +214,9 @@ async function injectViteIndexMiddleware(
 ) {
   const config = await getViteConfig();
 
-  app.get("/*", async (req, res, next) => {
+  app.use(Config.ignoreBase ? "/" : config.base, async (req, res, next) => {
```

Comment:
```suggestion
  app.use(config.base, async (req, res, next) => {
```

I think we can use `config.base` directly without introducing another value as it might be confusing. `config.base` was added to align `vite-express` with how Vite works with it, but as you noticed, it isn't used here, only in `injectStaticMiddleware` function. Also it's `/` by default that's why we dont need to check if its defined or not.

Reply:
Sure, using base directly in both functions will be more concise and align Vite Express with Vite.